www/src/components/package-readme.js

```diff
@@ -27,22 +27,35 @@ const PackageReadMe = props => {
         <a
           css={{
             "&&": {
-              display: githubUrl ? `inline-block` : `none`,
+              display: githubUrl ? `flex` : `none`,
               fontWeight: `normal`,
               border: 0,
               color: colors.gray.calm,
               boxShadow: `none`,
+              alignItems: `center`,
               "&:hover": {
                 background: `none`,
                 color: colors.gatsby,
               },
             },
           }}
           href={githubUrl}
-          aria-label={`${packageName} source`}
-          title={`View source on GitHub`}
+          aria-labelledby="github-link-label"
```

Comment:
For future reference, `aria-labelledby` isn't needed on a link if it contains text content.